bin/GoogleAdsCleanupServices.php

```diff
@@ -282,7 +299,7 @@ protected function remove_suffix( string $suffix, string $text ): string {
 	 * @param string $name
 	 */
 	protected function remove_use_statement( string $file, string $name ) {
-		$pattern = '/\s*use\s+[\w\d\\\]+' . preg_quote( $name, '/' ) . ';/';
+		$pattern = '/\s*use\s+[\w\d\\\]+\\\{1}' . preg_quote( $name, '/' ) . ';/';
```

Comment:
Fixes a bug when a service name is inside the name of another service

ie

For example, trying to delete `FeedItem` will match and delete as well `ExtensionFeedItem`. So we add the slash to be sure we don't delete undesired services.